src/components/views/dialogs/FeedbackDialog.js

```diff
@@ -100,6 +100,20 @@ export default (props) => {
         );
     }
 
+    let bugReports = null;
+    if (SdkConfig.get().bug_report_endpoint_url) {
+        bugReports = (
+            <p>{
+                _t("PRO TIP: If you start a bug, please submit <debugLogsLink>debug logs</debugLogsLink> " +
+                    "to help us track down the problem.", {}, {
+                    debugLogsLink: sub => (
+                        <AccessibleButton kind="link" onClick={onDebugLogsLinkClick}>{sub}</AccessibleButton>
+                    ),
+                })
+            }</p>
+        );
+    }
+
     return (<QuestionDialog
```

Comment:
what would this look like with neither a Countly nor Bug reports?

![image](https://user-images.githubusercontent.com/2403652/111389686-49c25680-86a9-11eb-9e80-5a9008f11220.png)

Just a heading?

Reply:
Countly is after the pro tip? It'd be just the search&report line

Reply:
Oh yes, github cut the diff in a misleading place